geo-postgis/Cargo.toml

```diff
@@ -12,4 +12,4 @@ edition = "2021"
 
 [dependencies]
 postgis = { version = ">=0.7.0, <0.9.0" }
-geo-types = { version = "0.8.0", path = "../geo-types" }
+geo-types = { version = "0.7", path = "../geo-types" }
```

Comment:
I've learned it's usually the wrong idea to try to bump versions in a PR - instead use the patches section.

We'll bump the version as part of the release process.